templates/citizen_project/landing.html.twig

```diff
@@ -119,6 +120,7 @@
                     {% else %}
                         <form id="start-creation-citizen-project" action="{{ path('app_adherent_create_citizen_project') }}" method="GET" class="b__nudge--top">
```

Comment:
what about change `action` attribute instead of adding the hidden input with path?
```
action={{ login_path_for_anonymous_follower('app_adherent_create_citizen_project') }}
```

Reply:
We cannot have the uri before submitting, and thus we would make another controller aware of the name of the field here.

Reply:
Sorry got confused, with the live discussion we had earlier :).

The real answer is that the form uses a get method, so on submission it removes previous params, loosing the one we need here.